src/Bicep.Core/Semantics/Namespaces/SystemNamespaceSymbol.cs

```diff
@@ -389,9 +404,113 @@ public class SystemNamespaceSymbol : NamespaceSymbol
                 .WithReturnType(LanguageConstants.String)
                 .WithDescription("Returns a value in the format of a globally unique identifier. **This function can only be used in the default value for a parameter**.")
                 .WithFlags(FunctionFlags.ParamDefaultsOnly)
+                .Build(),
+
+            new FunctionOverloadBuilder("loadTextContent")
+                .WithDescription("Load content of specified file into a string. Content loading occurs during compilation, not in runtime. Maximum allowed content size is 131,072 characters (including line endings).")
+                .WithRequiredParameter("filePath", LanguageConstants.String, "File's path which content will be loaded")
+                .WithOptionalParameter("encoding", LanguageConstants.String, "File encoding. If not provided, Bicep will try to automatically detect the encoding of a file based on the presence of byte order marks.")
+                .WithAdvancedReturnType(LanguageConstants.String, LoadTextContentTypeBuilder)
+                .WithExpressionEmitter(StringLiteralFunctionReturnTypeEmitter)
+                .Build(),
+
+            new FunctionOverloadBuilder("loadContentAsBase64")
+                .WithDescription($"Loads specified file as base64 string. File loading occurs during compilation, not in runtime. Maximum allowed size is {LanguageConstants.MaxLiteralCharacterLimit / 4 * 3 / 1024} Kb.")
+                .WithRequiredParameter("filePath", LanguageConstants.String, "File path which will be loaded")
```

Comment:
>"File path which will be loaded" [](http://example.com/codeflow?start=77&length=32)

Can we change to either "The file path" or "The path to the file that will be loaded"? Same comment above.

Reply:
done